doc/userguide/quickstart.rst

```diff
@@ -129,8 +129,8 @@ can be triggered quite easy. Before we trigger it, start ``tail`` to see updates
 
 Rule trigger::
 
-    sudo tail -f /var/log/suricata/fast.log
     curl http://testmyids.com/
+    sudo tail -f /var/log/suricata/fast.log
```

Comment:
The intent was to have `tail` running and then observe the alert added to `fast.log`.

I think this can be achieved by adding the `&` to the end of the `sudo tail ...` command.

Reply:
Yes that will work too.

Reply:
There's also the issue that testmyids.com doesn't work for this purpose anymore, it force redirects to https now.

Reply:
would it make sense that we as OISF provide a HTTP version without HTTPS for that?